src/components/DefineCompany/OfficeAddresses.vue

```diff
@@ -494,27 +495,25 @@ export default class OfficeAddresses extends Mixins(CommonMixin, EntityFilterMix
 </script>
 
 <style lang="scss" scoped>
-.address-list-container {
-  padding: 1rem;
-}
+@import '@/assets/styles/theme.scss';
 
 .meta-container {
   display: flex;
   flex-flow: column nowrap;
   position: relative;
 }
 
-.meta-container__inner {
-  margin-top: 1rem;
-}
-
 label:first-child {
   font-weight: 700;
   &__inner {
     flex: 1 1 auto;
   }
 }
 
+.meta-container__inner {
+  margin-top: 1rem;
```

Comment:
I would have moved this into the template, but there's a media query / styles for margin below, so I felt it was better to keep this here in this case.